### ISSUE TITLE
Disconnect and go back after reconnecting.

### ISSUE THREAD

**Issue by recoba19** [stackblitz/bolt.new#3981]
### Describe the bug

I'd been working on a project for a while, but when I tried to add a photo as an attachment, I got a window to log me back in.
I reconnected, and the project loaded in a very old version. I lost more than an hour's work, even though I had a lot of changes to make.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-9d7lq3js

### Steps to reproduce

After a modification, an image is added as an attachment.
Window to reconnect.
Reconnect
Back to my project
Loading the project in a very old version with all changes lost.

### Expected behavior

I'm waiting to be able to retrieve the latest version of my project after reconnecting.

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 67149101,
  "usedJSHeapSize": 56515489,
  "jsHeapSizeLimit": 4294705152
}
Username = recoba19
Chat ID = 45ef85f66718
```

**Comment by skadayam** [comment]
Yes, this happened to me today too - I have lost many hours of work too. For me, I encountered strange behaviors in Bolt, and I decided to close the project (did so by choosing to open a new conversation from the left sidebar in Bolt). Then chose my working project from the left side bar. I entered in to find that the state of the code I had was not the same as it was just a minute ago - it was older. I thought I must have made some mistake - until I read this bug report. Thank you.
